src/components/BaseHeading/BaseHeading.css

```
@@ -0,0 +1,37 @@
+.base-heading {
+  font-family: 'Plus Jakarta Sans', sans-serif;
+  color: var(--color-black);
+}
+
+.base-heading--xl {
+  font-size: 24px;
+  line-height: 30px;
+}
+
+.base-heading--l {
+  font-size: 18px;
+  line-height: 23px;
+}
+
+.base-heading--m {
```

Ajustar los `modifiers` de las clases acorde a los nuevos valores del `size`, ejemplo:

```suggestion
.base-heading--md {
```